download.md

```diff
@@ -151,50 +151,32 @@ function submit_download() {
 }
 
 function get_form_data() {
-    var bidders = [];
-    var analytics = [];
+    var modules = [];
     var version = $('.selectpicker').val();
     var removedModules = [];
 
-    var version_restrictions=$('.version-restriction');
-    for (var i = 0; i < version_restrictions.length; i++) {
-        var module=version_restrictions[i].getAttribute('moduleCode');
-        var restriction_name=version_restrictions[i].getAttribute('name');
-        var restriction_value=version_restrictions[i].getAttribute('value');
-        if (restriction_name == "min-version") {
-          var module_version_array=restriction_value.split(".");
-          var pbjs_version_array=version.split(".");
+    // bracket escape is to workaround markdownlint false positive
```

Comment:
Oh boy :smile:  What do you think of putting this file under [assets/js](https://github.com/prebid/prebid.github.io/tree/master/assets/js) and add it as a `script` tag instead?

Reply:
I moved the JavaScript. I also removed the hidden modal, which hasn't been working since forever.